projects/binutils/build.sh

```diff
@@ -16,6 +16,7 @@
 ################################################################################
 
 # build project
+export UBSAN_OPTIONS=silence_unsigned_overflow=1
```

Comment:
No you need to change CFLAGS, CXXFLAGS with -fno-sanitize=<whatever>. This change only works during build process, and not for runtime.

Reply:
Thanks, I reused sample code from libxslt. Is it correct now ?